proposals/bring_your_own_argo_workflows/byoaw_test_plan.md

```diff
@@ -0,0 +1,305 @@
+
+### Out of Scope
+- Partial ArgoWF installs combined with DSP-shipped Workflow Controller
+- Isolation between DSP ArgoWF WC and vanilla cluster-scoped ArgoWF installation
+
+## Test Environment Requirements
+
+### Prerequisites
+- OpenShift/Kubernetes clusters with ODH/DSP installed
+- Multiple test environments with different Argo Workflows versions
+- Access to modify DataScienceCluster and DSPA configurations
+- Sample pipelines covering various complexity levels
+- Test data for migration scenarios
+
+### Test Environments
+| Environment | Argo Version   | DSP Version | Purpose                       |
+|-------------|----------------|-------------|-------------------------------|
+| Env-1       | Current(3.7.x) | Current     | N version compatibility       |
+| Env-2       | 3.6.x          | Current     | N-1 version compatibility     |
+| Env-3       | 3.4.x - 3.5.y  | Previous    | Upgrade scenarios             |
+
+## Test Categories
+
+## 1. Cluster Configuration Tests
+This section covers tests for different cluster configurations to ensure BYOAW functionality across various deployment scenarios.
+
+### 1.1 Global Configuration Toggle
+
+| Test Case ID          | TC-CC-001                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify global toggle to disable WorkflowControllers works correctly                                                                                                                                                                                                                                                                                                                                                                         |
+| **Test Steps**        | <ol><li>Install ODH with default configuration (WorkflowControllers enabled)<br/>1. Create DSPA and verify WorkflowController deployment<br/>1. Update DataScienceCluster to disable WorkflowControllers:<br/>`spec.components.datasciencepipelines.argoWorkflowsControllers.managementState: Removed`<br/>1. Verify existing WorkflowControllers are removed<br/>1. Create new DSPA and verify no WorkflowController is deployed</li></ol> |
```

Comment:
The numbered list markdown isn't parsing correctly here (1. step1 <br />1. step2).  From later tables it looks like you need to use raw html tags (ie <ol><li>step1</li><li>step2</li></ol>